### PR TITLE
Add Dockefile

### PR DESCRIPTION
**Pull request by netflash**
Hello there,

I find useful to have a Dockerfile - it's more convenient to run this tool on a machine which doesn't have golang installed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,4 +60,19 @@ Once built, you can run remote commands like this:
 ./winrm -hostname remote.domain.com -username "Administrator" -password "secret" "ipconfig /all"
 ```
 
+## Docker
+
+### Build container
```
Comment (xrstf) [inline README.md]: This should read "Build image".

File: README.md [hunk 2]
```diff
@@ -60,4 +60,19 @@ Once built, you can run remote commands like this:
...
+```
+docker build -t winrm .
+```
+
+### Usage
+
+Once built, you can run remote commands like this:
+
+```sh
+docker run -it --rm winrm winrm -hostname remote.domain.com -username "Administrator" -password "secret" "ipconfig /all"
```
Comment (xrstf) [inline README.md]: Given that the image is meant to run the winrm client, one could set the winrm binary as the image's entrypoint.

**Comment by masterzen** [comment]
It looks like I've never answered this PR, and @xrstf comment somehow reminded me to do it, so I'm very sorry for the lack of answers.

I can accept this PR, but provided the commit history is squashed with a proper commit message (why is there a merge commit there?) and please fix the very valid @xrstf comments (especially the entrypoint one).

Thanks,

**Comment by netflash** [comment]
@masterzen hey
I've updated the PR. As for the merge commit - I had it first peer reviewed internally, that's what the merge comes from. No harm here.

**Comment by netflash** [comment]
@masterzen @xrstf ping

**Comment by masterzen** [comment]
Hi @netflash, 

Thanks for pinging me to merge this PR, I'm prone to forget to follow-up :)
I'm OK to merge it ASAP, but would you mind squashing all the commits in one commit ?

Thanks!

**Comment by netflash** [comment]
@masterzen done 
better late than never :-)

**Comment by masterzen** [comment]
@netflash thanks for your contribution!

**Pull request merged**